niworkflows/viz/utils.py

```diff
@@ -56,7 +59,7 @@ def as_svg(image):
 
 def cuts_from_bbox(mask_nii, cuts=3):
     from nibabel.affines import apply_affine
-    imnii = nb.load(mask_nii)
+    imnii = mask_nii
```

Comment:
I understand, but then please update the whole function to avoid that double referencing, right? 
--
Could we have a nilearn-like behavior, and make this function allow both file paths and nifti images as input?